docs/run-local.md

```diff
@@ -59,7 +59,7 @@ In order to launch or update the system to the latest built images or chosen con
 
 ```bash
 ./launch-or-update-local.sh
-# or to run without getting the latest development chart versions (i.e. run with versions that are currently set)
+# or to run without getting the latest development chart versions (i.e. run with versions that are coming from builds on main branches)
```

Comment:
Can we please keep this to master for now, so it's consistent with the current branch names. I think master / main is something we should cover as a separate activity.